src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -48,6 +51,8 @@ interface BuildLocator {
      */
     fun withAllBranches() : BuildLocator
 
+    fun withSinceBuild(locator: BuildLocator): BuildLocator
```

Comment:
`withSinceBuild` doesn't look grammatically correct to me, why not `sinceBuild`?
And I don't understand how it's supposed to work. BuildLocator may return two or more build, how `sinceBuild` will work in such cases? And what it'll return if `locator` finds no builds? I suppose that anyone who want to use sinceBuild option will firstly need to find a starting build and somehow handle cases when there are no ore more then one builds. So why not make this method take `BuildId` as an argument?